src/Files.App/ViewModels/HomeViewModel.cs

```diff
@@ -181,9 +181,10 @@ public void RemoveWidget<TWidget>() where TWidget : IWidgetViewModel
 
 		// Command methods
 
-		private void ExecuteHomePageLoadedCommand(RoutedEventArgs? e)
+		private async void ExecuteHomePageLoadedCommand(RoutedEventArgs? e)
 		{
 			ReloadWidgets();
+			await RefreshWidgetProperties();
```

Comment:
Will this be executed if you switch between two open tabs?

Reply:
It won't